### PR TITLE
Create intent for Chrome without the need of timeout()

### PR DESCRIPTION
**Pull request by aamirp**
On Android and v 5.0 and above Chrome magically changes the referrer to com.android.chrome which means we do not get paid for Web-Installs when using Chrome on Android 5.0+  

Followed this comment
https://code.google.com/p/chromium/issues/detail?id=459711&thanks=459711&ts=1424288965
and made a fix for it. Tested and it seems to work. 

There was no need to test for OS Version in Javascript since it seemed to work with OS version 4.2 as well.

### CODE REVIEW

**Comment by andrewlook** [comment]
DUDE! this is awesome :) great find!

File: urx.js [hunk 1]
```diff
@@ -178,9 +178,9 @@ $urx = (function(window) {
         window.location = desktopLink;
     }
 
-    var androidDeeplinkLaunch = function(deeplink, appId, timeoutCallback) {
-        // If no appId was given, this is actually a web link
-        if (appId === undefined) {
+    var androidDeeplinkLaunch = function(installLink, deeplink, appId, timeoutCallback) {
+        // If no appId was given, this is actually a web link        
+        if (appId === undefined) {
```
Comment (andrewlook) [inline urx.js]: can we clean up some of this trailing whitespace before committing please?

File: urx.js [hunk 2]
```diff
@@ -199,22 +199,22 @@ $urx = (function(window) {
...
+                // When deeplinking on chrome 35+, there is inherent app store fallback logic built into the browser (likely a bug in Chrome)
+                // Workaround for Bug https://code.google.com/p/chromium/issues/detail?id=459711&thanks=459711&ts=1424288965.
+                window.location.replace("intent:"+intent+"#Intent;scheme="+scheme+";package="+appId+";S.browser_fallback_url="+installLink+";end");
             }
-            // When deeplinking on chrome 35+, there is inherent app store fallback logic built into the browser (likely a bug in Chrome)
-        } else {
-            timeoutCallback();
-        }
+            
+         }
```
Comment (andrewlook) [inline urx.js]: looks like the indentation is a little bit funny here

File: urx.js [hunk 3]
```diff
@@ -475,7 +475,7 @@ $urx = (function(window) {
                             };
                             searchResult.resolveDeeplinkWithAppStoreFallback = function(failure) {
                                 this.resolveDeeplink(failure, function(resolutionData){
-                                    if (resolutionData.installLink !== undefined) {
+                                    if (resolutionData.installLink !== undefined) {
```
Comment (andrewlook) [inline urx.js]: trailing whitespace

**Comment by jlturner** [comment]
:+1: Awesome find on the referrer link!

**Comment by jlturner** [comment]
Rebased and merged into master.

**Comment by jlturner** [comment]
Release Tag v1.1 cut for this update.

**Pull request closed without merging**